examples/research/BLIP-1.md

```diff
@@ -0,0 +1,35 @@
+# Use of ZKP, CCSM Anchoring, and Verifiable Credentials to Create a Trustless Supply Chain. 
+
+## Summary
+
+This document contains research derived from the BLIP-1 and BLIP-14 efforts. These research efforts focus on determining when the use of zero knowledge proofs and anchoring state changes on a CCSM are appropriate for multi-party workflow steps. The resources listed below outline a multi-party international supply chain use case and the business & technical requirements that provided the frame for this analysis. Additionally, the requirements outlined in the documentation provided provide a foundation for which an international supply chain demo application can be implemented. 
+
+These resources illustrate how supply chain documents (state objects) can be:
+* exchanged in zero trust and under zero knowledge,
+* synchronized across multiple parties,
+* and verified by third parties to be correct. 
+
+Technical elements highlighted in the following resources are:
+* The use of Verifiable Credentials as state objects representing Supply Chain Documents.
+* The need for CCSM's to provide provable, tamper resistant timestamping for state changes.
+* The use of ZKP's to provide automated validation and/or privacy.
+
+## Blog 
+
+* Upcoming*
+
+## Resources
+* [Intl. Supply Chain Sequence Diagram](https://drive.google.com/file/d/1wPzEjGPr0QvHqF7O3ei3Wns28dVaQDX3/view?usp=sharing)
```

Comment:
small thing, but i think diagrams should be part of the repo and then you can use relative link here

edit: ok i see there are some docs too, i think it would be better if we have them all in repo, or in some official docs like gitbook, but this should be fine too i guess

Reply:
@markymark-26 please create a folder with the diagrams and do a relative reference in the text. We should not reply on non-permanent external links.